co-log-benchmark/Main.hs

```diff
@@ -0,0 +1,103 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Main
+  ( main
+  ) where
+
+import Data.Time.Clock
```

Comment:
```suggestion
import Data.Semigroup ((<>))
import Data.Time.Clock
```

Reply:
I hope that should fix CI error. But the last commit with `base-noprelude` should be reverted. Alternative, you can add `Prelude` module to the `co-log-benchmarks` packages